### PR TITLE
test: 💍 acceptance tests for injected credentials

### PR DESCRIPTION
**Pull request by DhariniJeeva**
[Jira](https://hashicorp.atlassian.net/browse/ICU-5572)

### CODE REVIEW

**Review by cameronperera — APPROVED** [review]

File: ui/admin/tests/acceptance/targets/injected-application-credential-sources-test.js [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+      instances.credential = instances.credentials[0];
+      instances.target = this.server.create('target', {
+        scope: instances.scopes.project,
+        type: 'ssh',
+      });
+
+      randomlySelectedCredentials = this.server.schema.credentials
+        .all()
+        .models.filter(Boolean)
+        .map((cred) => cred.id);
```
Comment (ZedLi) [inline ui/admin/tests/acceptance/targets/injected-application-credential-sources-test.js:87]: What does the `.models` do? Doesn't `.all()` return models?

Would this work? 
```suggestion
      randomlySelectedCredentials = this.server.schema.credentials
        .where(Boolean)
        .map((cred) => cred.id);
```

**Review by ZedLi — APPROVED** [review]
LGTM, just had one question

File: ui/admin/tests/acceptance/targets/injected-application-credential-sources-test.js (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/tests/acceptance/targets/injected-application-credential-sources-test.js:87]: ah, I believe this is a copy/paste error. let me clean this up

Comment (DhariniJeeva) [reply ui/admin/tests/acceptance/targets/injected-application-credential-sources-test.js]: I removed the filter part as that is not needed

**Pull request merged**